app/services/socket-io.js

```diff
@@ -26,17 +26,19 @@ export default Service.extend({
   * url meaning multiple requests for the same socket will return the same object.
   */
   socketFor(url, options = {}) {
-    const existingProxy = this.get(`sockets.${normalizeURL(url)}`);
+    const normalizedUrl = normalizeURL(url);
+    const cleanedUrl = normalizedUrl.replace('.', '', 'g');
```

Comment:
@thoov this is not working on Chrome 48, this method of specify flag is being deprecated - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

Instead:

    const cleanedUrl = normalizedUrl.replace(/\./g, '');

Otherwise, thanks for this addon - it's great!